src/Components/NavBar/NavBar.tsx

```diff
@@ -11,10 +12,10 @@ const NavBar: React.FC = () => {
       <nav>
         <ul className='menu'>
             <li>
-              <NavLink to='/' className='selected' id="dashboard-nav-button">Dashboard</NavLink>
+              <NavLink to='/' style={({ isActive }) => isActive ? {color: '#a17f1a'}:{}} id="dashboard-nav-button">Dashboard</NavLink>
             </li>
             <li>
-              <NavLink to='/writing-page' className='selected' id="start-writing-nav-button">Start Writing</NavLink>
+              <NavLink to='/writing-page' style={({ isActive }) => isActive ? {color: '#a17f1a'}:{}} id="start-writing-nav-button">Start Writing</NavLink>
```

Comment:
kind of want to know what this is doing when you get a chance this seems new to me

Reply:
Very cool. I like the use of an active state for styling the current link